src/app/core/spinner-service/spinner.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SpinnerService {
+  private loadingBehSub= new BehaviorSubject<boolean>(false);
+  spinner$!:Observable<boolean>;
+
+  constructor() {
+    this.spinner$=this.loadingBehSub.asObservable();
```

Comment:
todo esto esta bien, pero vamos a hacer las cosas un poco diferente esta vez.
en vez tener una propiedad spinner$ lo que vamos a hacer es agregar un methodo get que retorne this.loadingBehSub.asObservable();

Reply:
elimina spinner$ propiedad